tests/src/lib/event_mapper/test_emr_serverless_event_mapper.py

```diff
@@ -0,0 +1,163 @@
+import pytest
+from unittest.mock import patch, MagicMock
+from datetime import datetime
+
+from lib.event_mapper import EMRServerlessEventMapper, EMRServerlessEventMapperException
+from lib.core.constants import EventResult, SettingConfigResourceTypes as types
+
+EVENT_TYPE = "EMR Serverless Job Run State Change"
+EMR_APP_NAME = "emr-app-test"
+EMR_APP_ID = "00fkm6vigfuq6215"
+JOB_NAME = "emr-job-test"
+JOB_RUN = {
+    "jobRun": {
+        "applicationId": EMR_APP_ID,
+        "jobRunId": "00flfo8g80vlko17",
+        "name": JOB_NAME,
+        "createdAt": str(datetime(2000, 1, 1, 0, 0, 0)),
+        "updatedAt": str(datetime(2000, 1, 1, 1, 0, 0)),
+        "state": "FAILED",
+        "stateDetails": "Job failed",
+        "jobDriver": {
+            "sparkSubmit": {
+                "entryPoint": "s3://s3-salmonts-emr-scripts-dev/job2_failure.py"
```

Comment:
I guess, s3 bucket name doesn't matter here, so maybe let's use something "not real"

Reply:
I'm wondering if it makes sense to add EMR serverless items into sample config

Reply:
sure, let me update this entryPoint and sample config
thank you for your comments!